tests/scripts/helm.sh

```diff
@@ -26,19 +24,31 @@ detectArch() {
     esac
 }
 
-install() {
-    if ! helm_loc="$(type -p "helm")" || [[ -z ${helm_loc} ]]; then
-        # Download and unpack helm
-        local dist
-        dist="$(uname -s)"
+# Echo the name or path of the helm command. Install helm to a temp location
+# if it's not present.
+install_helm() {
+    local helm="helm"
+    local dist="$(uname -s)"
+    dist="$(echo "${dist}" | tr '[A-Z]' '[a-z]')"
+    local tmp_helm="${temp}/${dist}-${arch}/helm"
+
+    if [[ -f "${tmp_helm}" ]]; then # helm installed by this script
+        helm="${tmp_helm}"
+    elif ! which ${helm} &>/dev/null; then
+        echo "Installing helm..." >&2
         # shellcheck disable=SC2021
-        dist=$(echo "${dist}" | tr "[A-Z]" "[a-z]")
         mkdir -p "${temp}"
-        wget "https://storage.googleapis.com/kubernetes-helm/helm-v2.13.1-${dist}-${arch}.tar.gz" -O "${temp}/helm.tar.gz"
-        tar -C "${temp}" -zxvf "${temp}/helm.tar.gz"
-        HELM="${temp}/${dist}-${arch}/helm"
+	local helm_url="https://storage.googleapis.com/kubernetes-helm/helm-v2.13.1-${dist}-${arch}.tar.gz"
```

Comment:
The indentation is wrong here. It has to be one further in.

Reply:
right, mistakenly used a tab here and below.